bin/martinize2

```diff
@@ -963,24 +942,24 @@ def entry():
         node_selector = node_selectors[args.posres]
         vermouth.ApplyPosres(node_selector, args.posres_fc).run_system(system)
 
-    if args.govs_includes:
-        # The way Virtual Site GoMartini works has to be in sync with
-        # Sebastian's create_goVirt.py script, until the method is fully
-        # implemented in vermouth. One call of martinize2 must create a single
-        # molecule, regardless of the number of fragments in the input.
-        # The molecule type name is provided as an input with the -govs-moltype
-        # flag to be consistent with the name provided to Sebastian's script.
-        # The name cannot be guessed because a system may need to be composed
-        # from multiple calls to martinize2 and create_goVirt.py.
-        LOGGER.info("Adding includes for Virtual Site Go Martini.", type="step")
-        LOGGER.info(
-            "The output topology will require files generated by " '"create_goVirt.py".'
-        )
-        vermouth.MergeAllMolecules().run_system(system)
-        vermouth.SetMoleculeMeta(moltype=args.govs_moltype).run_system(system)
-        vermouth.GoVirtIncludes().run_system(system)
+    # Generate the Go model if required
+    if args.go:
+        LOGGER.info("Reading Go model contact map.", type="step")
+        go_map = read_go_map(args.go)
+        LOGGER.info("Generating the Go model.", type="step")
+        GoPipeline.run_system(system,
+                              moltype=args.govs_moltype,
+                              contact_map=go_map,
+                              cutoff_short=args.go_low,
+                              cutoff_long=args.go_up,
+                              go_eps=args.go_eps,
+                              res_dist=args.go_res_dist,)
```

Comment:
I missed this in an earlier review, but these arguments must be arguments to the GoPipeline rather than the run_system method!

Reply:
No this is on purpose. They are given to `run_system` such that you can define the pipeline object without having to know the particular arguments given to it (i.e. `GoPipeline = PiplineObject(args_known_before)`).  In my opinion, this improves the understandability of the code

Reply:
As a counterargument it breaks with all Processor conventions we've had for a few years now